### PR TITLE
Deprecate charts

### PR DESCRIPTION
**Pull request by Zelldon**
Deprecates the zeebe-* charts

Add note on:

 * zeebe-full
 * zeebe-cluster
 * zeebe-operate
 * zeebe-tasklist

Marks them depreacted in the chart.yaml

### CODE REVIEW

File: charts/zeebe-cluster-helm/templates/NOTES.txt [hunk 1]
```diff
@@ -20,4 +20,16 @@ The Cluster itself is not exposed as a service that means that you can use kubec
...
+ | |  | | | |__    | |__) | | |__) | | |__    | |         /  \       | |    | |__    | |  | |
+ | |  | | |  __|   |  ___/  |  _  /  |  __|   | |        / /\ \      | |    |  __|   | |  | |
+ | |__| | | |____  | |      | | \ \  | |____  | |____   / ____ \     | |    | |____  | |__| |
+ |_____/  |______| |_|      |_|  \_\ |______|  \_____| /_/    \_\    |_|    |______| |_____/
+
+
+Please be aware that this helm chart has been deprecated and is no longer maintained!
+It was fully replaced by new helm chart, check out helm.camunda.io for more details.
+
+
```
Comment (menski) [inline charts/zeebe-cluster-helm/templates/NOTES.txt:34]: the other deprecation notes don't have empty lines, we should align this. I was think that empty lines could better highlight the note and make it more visible in terminal, what do you think?

File: charts/zeebe-cluster-helm/templates/NOTES.txt [hunk 2]
```diff
@@ -20,4 +20,16 @@ The Cluster itself is not exposed as a service that means that you can use kubec
...
+  _____    ______   _____    _____    ______    _____              _______   ______   _____
+ |  __ \  |  ____| |  __ \  |  __ \  |  ____|  / ____|     /\     |__   __| |  ____| |  __ \
+ | |  | | | |__    | |__) | | |__) | | |__    | |         /  \       | |    | |__    | |  | |
+ | |  | | |  __|   |  ___/  |  _  /  |  __|   | |        / /\ \      | |    |  __|   | |  | |
+ | |__| | | |____  | |      | | \ \  | |____  | |____   / ____ \     | |    | |____  | |__| |
+ |_____/  |______| |_|      |_|  \_\ |______|  \_____| /_/    \_\    |_|    |______| |_____/
+
+
+Please be aware that this helm chart has been deprecated and is no longer maintained!
+It was fully replaced by new helm chart, check out helm.camunda.io for more details.
```
Comment (menski) [inline charts/zeebe-cluster-helm/templates/NOTES.txt:32]: Proposal for the note:

> Please note that this helm chart has been deprecated and is no longer maintained!
> Check out helm.camunda.io to find more information on how to upgrade to the new helm chart.

**Review by menski — CHANGES_REQUESTED** [review]
Probably out of scope but what are we doing with `zeebe-zeeqs-helm`?

File: charts/zeebe-cluster-helm/templates/NOTES.txt (hunk 1 above)
Comment (Zelldon) [reply charts/zeebe-cluster-helm/templates/NOTES.txt:34]: So one addition line or more?

**Comment by Zelldon** [comment]
Discussed with menski the review hints and one line extra is good and he approves.

**Pull request merged**